at-quannguyen/W4/ex2.rb

```diff
@@ -0,0 +1,30 @@
+def add_more(num)
+  ->(x) { x + num }
+end
+
+def multi_more(num)
+  ->(x) { x * num }
+end
+
+def compose(proc1, proc2)
+  ->(x) { proc2.(proc1.(x)) }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/cd24ce24c04c57d4b75cc24b8a37d518f57e5e0e/at-quannguyen/W4/ex2.rb#L10"></span>Style/LambdaCall: Prefer the use of <code>lambda.call(...)</code> over <code>lambda.(...)</code>. (https://github.com/rubocop-hq/ruby-style-guide#proc-call)</td>
    </tr>
  </tbody>
</table>